libs/ui/layout/src/lib/value-list/value-list.component.html

```diff
@@ -0,0 +1,44 @@
+<div
+  class="flex items-center justify-center"
+  cdkOverlayOrigin
+  #trigger="cdkOverlayOrigin"
+  [class]="extraClass"
+>
+  <gn-ui-button
+    #iconElement
+    type="light"
+    extraClass="flex items-center bg-transparent border-none justify-center"
+    (buttonClick)="toggleOverlay()"
+  >
+    <ng-icon name="iconoirList" size="24"></ng-icon>
+  </gn-ui-button>
+
+  <ng-template
```

Comment:
not align with the  right border
![image](https://github.com/user-attachments/assets/0fa299eb-38e7-46eb-9560-8efe7d0ae241)